internal/sgbouncewizard/sgbouncewizard.go

```diff
@@ -161,20 +161,30 @@ func (srv *Server) GetAllRulesRoute(w http.ResponseWriter, r *http.Request) {
 	}
 
 
-	limit, err := strconv.Atoi(r.FormValue("limit"))
+	if (len(offset_param) == 0 &&  len(limit_param) == 0){
+		limit := math.MaxInt32
```

Comment:
Instead we should call and return `srv.DBClient.GetAllRules()` and implement a new function `srv.DBClient.GetRules()` that takes in an offset and limit, which would be called below on line 190